Updates

jspaith opened the pull request:
Make easier to copy/paste and more critically clarify certificate naming practices.

azurecla commented:
Hi __@alexandair__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

PRmerger commented:
@alexandair : Thanks for your contribution to the Azure documentation! The author, @kundanap, has been notified to review your proposed change.

cynthn commented:
@alexandair thank you for the contribution! This file has been rewritten since you made your proposed changes and the fixes were resolved in that rewrite. I am going to ask the PR mergers to close this PR. Thank you again!

jspaith committed:
Updates

Make easier to copy/paste and more critically clarify certificate naming practices.
